src/Shell/Task/ModelTask.php

```diff
@@ -1031,14 +1031,14 @@ protected function _getAllTables()
                 'Connections need to implement schemaCollection() to be used with bake.'
             );
 
-            return $this->_stop();
+            $this->_stop();
```

Comment:
You might need this return for when stop is mocked. You could also use abort() instead.